ethpm_cli/parser.py

```diff
@@ -86,22 +90,25 @@ def install_action(args: argparse.Namespace) -> None:
     )
 
 
-install_parser = ethpm_parser.add_parser("install", help="Install uri")
+install_parser = ethpm_parser.add_parser(
+    "install",
+    help="Install a target package, by providing its uri, to your ethPM directory.",
+)
 install_parser.add_argument(
     "uri",
     action="store",
     type=str,
-    help="IPFS / Github / Registry URI of package you want to install.",
+    help="IPFS / Github / Etherscan / Registry URI of target package.",
 )
 install_parser.add_argument(
     "--ethpm-dir",
     dest="ethpm_dir",
     action="store",
     type=Path,
-    help="Path to specific ethpm_packages dir.",
+    help="Path to specific ethPM directory (Defaults to ``cwd/_ethpm_packages``).",
```

Comment:
Maybe instead of `cwd/_ethpm_packages` this could be `./_ethpm_packages`.  Alternatively, have you looked at how any other common CLI tools denote this?